templates/config/overlays/namespaced/role-binding.json

```diff
@@ -0,0 +1,3 @@
+[{"op": "replace", "path": "/kind", "value": "RoleBinding"},
+{"op": "add", "path": "/metadata/namespace", "value":  "ack-system"},
```

Comment:
This is replacing the namespace that should be supplied by the user in `.Values.watchNamespace` with the hard-coded string "ack-system", which is defeating the original purpose of a namespace-scoped installation, no?

Reply:
This file is for the Kustomize overlay - no Helm values to replace, here. All of these Kustomize files use `ack-system` as the default namespace. The user will need to override the values for all resources anyway, so may as well make this one match.